Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs

```diff
@@ -229,8 +229,8 @@ private void InjectDoAfter(InjectorComponent component, EntityUid user, EntityUi
 
         var actualDelay = MathF.Max(component.Delay, 1f);
 
-        // Injections take 1 second longer per additional 5u
-        actualDelay += (float) component.TransferAmount / component.Delay - 1;
+        // Injections take 0.5 seconds longer per additional 5u
+        actualDelay += (float) component.TransferAmount / component.Delay - 0.5f;
```

Comment:
I don't think decreasing time will affect that much

Reply:
Not really; but you're looking at a difference of 3 -> 1.5 seconds on 15u
It mostly only affects large doses from the syringe.